modules/aws_dns/aws-dns.yaml

```diff
@@ -42,6 +42,11 @@ inputs:
     user_facing: false
     description: Deprecated
     default: ""
+  - name: linked_module
+    user_facing: true
+    validator: str(required=False)
+    description: The module type (or name if given) to automatically add root dns records for.
+    default: ""
```

Comment:
Let's make default `k8s-base` instead of handling None in the code

Reply:
Valid point, but these defaults aren't exposed in the code-- they're only filled in once terraform generation is done.

Reply:
oh, ok.